roles/openshift_prometheus/templates/prometheus.j2

```diff
@@ -104,7 +104,7 @@ spec:
         - mountPath: /prometheus
           name: prometheus-data
 
-      # Deploy alertmanager behind prometheus-alert-buffer behind an oauth proxy
+      # Deploy alerts behind alerts-proxy behind an oauth proxy
```

Comment:
`Deploy alert-buffer behind alerts-proxy` is clearer

Reply:
ok